data/tools/extract_cldr_data.py

```diff
@@ -0,0 +1,84 @@
+"""
+Extracts data from CLDR-JSON repository, e.g. nouns like month or day names.
+Script either creates a new inflection file, or appends data to existing one.
+
+Part of Speech, Gender, Case, Number. Animacy
+
+Run script from data folder.
+
+Before running the script clone cldr-json repository:
+
+gh repo clone unicode-org/cldr-json
+
+and install jsonpath-ng package:
+
+pip install jsonpath-ng
+"""
+
+import argparse
+import json
+import os
+
+from jsonpath_ng import jsonpath, parse
+
+
+def load_json(filename):
+    """Loads JSON data from the specified file.
+
+    Args:
+        filename: The name of the JSON file.
+
+    Returns:
+        The parsed JSON data.
+    """
+
+    try:
+        with open(filename, 'r', encoding='utf-8') as file:
+            return json.load(file)
+    except FileNotFoundError:
+        print(f"Error: File '{filename}' not found.")
+        return None
+
+
+def write_to_lexicon(output_file, language, json_data):
+    """Extracts specified data from cldr-json file
+       and writes it to the lexicon file.
+
+    Args:
+        output_file: name of the lexicon.
+        language: cldr-json file language.
+        json_data: parsed cldr-json data.
+    """
+    MONTH_NAMES_EXPRESSION = parse('main..dates.calendars.gregorian.months.format.wide.*')
+    DAY_NAMES_EXPRESSION = parse('main..dates.calendars.gregorian.days.format.wide.*')
+    EXPRESSIONS = [MONTH_NAMES_EXPRESSION, DAY_NAMES_EXPRESSION]
+
+    results = []
+    for expression in EXPRESSIONS:
+        match = expression.find(json_data)
+        for m in match:
+            results.append(m.value + ': noun masculine nominative singular inanimate\n')
```

Comment:
This doesn't match what I see in the files above.  Are you hand-editing them after the fact, or am I misunderstanding something?

Reply:
I generate files with all the grammatical forms, but then delete unnecessary information (depending on the language). So there's no bug - we can expand the tool to know more about each language in the future, and reduce manual work. I leave that to the reader :).

Reply:
Ah, so you _do_ hand-edit.  I wasn't objecting; I was just worried I'd missed something.  :-)